administration/src/bp-modules/stores/StoresTable.tsx

```diff
@@ -4,13 +4,13 @@ import { useCallback } from 'react'
 import styled from 'styled-components'
 
 import { StoreFieldConfig } from '../../project-configs/getProjectConfig'
+import dimensions from '../constants/dimensions'
 import { AcceptingStoreEntry } from './AcceptingStoreEntry'
 
 const TableContainer = styled.div`
   width: 100vw;
-  height: 85vh;
+  height: calc(100vh - ${dimensions.navigationBarHeight + dimensions.bottomBarHeight}px);
```

Comment:
i'm not super happy with that solution. But we need a height to get the container scrollable and to keep the BottomBar sticky shown below. If someone has a better solution feel free to propose